pkg/controller/gkenetworkparamset/gkenetworkparamsetmetrics.go

```diff
@@ -0,0 +1,32 @@
+package gkenetworkparamset
+
+import (
+	"sync"
+
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+// GKENetworkParamSetSubsystem - subsystem name used for GKE Network Param Sets
+const GKENetworkParamSetSubsystem = "gkenetworkparamset_controller"
+
+var (
+	FetchSubnetErrs = metrics.NewCounterVec(
```

Comment:
do we need to export this variables? shouldn't they be all private (lowercase)?

Reply:
I was actually not aware that starting a variable with a capital letter would export it. I was following a similar convention to https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/endpointslice/metrics/metrics.go when writing this file. Do we not need to follow the same style in this case since the metrics are in the same package as the controller?

Reply:
oh, yeah, in golang uppercase exports them, the example linked need to export them because they are used in a different package. Since we are using them locally and if you don't think we need to use it in another package, is always better to keep them local and not export them